<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionsWindowOne</class>
 <widget class="QDialog" name="QuestionsWindowOne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>821</width>
         <height>491</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>801</width>
          <height>91</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>60</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="previous">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>450</y>
          <width>92</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>60</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Time remaining:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="next">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>450</y>
          <width>92</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QPushButton" name="finish">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>450</y>
          <width>92</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>721</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>701</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>701</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>90</y>
           <width>701</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>120</y>
           <width>701</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
